src/containers/ModelPanel/index.tsx

```diff
@@ -114,11 +130,47 @@ const ModelPanel: React.FC<ModelPanelProps> = ({
         ),
     };
 
+    const handleRestorDefaultClick = () => {
+        clearUserSelectedColors();
+        setCurrentColorSettings({
+            currentColorSettings: ColorSettings.Default,
+        });
+    };
+
+    const handleRestoreDefaultMouseEnter = () => {
+        if (!defaultUiSettingsApplied) {
+            setCurrentColorSettings({
+                currentColorSettings: ColorSettings.Default,
+            });
+        }
+    };
+
+    const handleRestoreDefaultMouseLeave = () => {
+        if (!defaultUiSettingsApplied) {
+            setCurrentColorSettings({
+                currentColorSettings: ColorSettings.UserSelected,
+            });
+        }
+    };
+
     return (
         <div className={styles.container}>
             <SideBarContents
                 mainTitle="Agents"
-                content={[contentMap[viewerStatus]]}
+                content={[
+                    <div className={styles.container} key="molecules">
+                        {contentMap[viewerStatus]}
+                        <NavButton
+                            titleText={"Restore color defaults"}
+                            buttonType={ButtonClass.Action}
+                            isDisabled={defaultUiSettingsApplied}
+                            clickHandler={handleRestorDefaultClick}
+                            onMouseEnter={handleRestoreDefaultMouseEnter}
+                            onMouseLeave={handleRestoreDefaultMouseLeave}
+                        />
+                    </div>,
+                    null,
+                ]}
```

Comment:
This is the only new piece of UI work, a button to preview and restore default settings that is active when user selected settings are currently applied.